node/src/components/consensus/utils.rs

```diff
@@ -24,21 +24,26 @@ pub(crate) fn validate_finality_signatures(
     Ok(())
 }
 
-/// Computes the lower bound for the fraction of weight of signatures that will be considered
+/// Computes the quorum for the fraction of weight of signatures that will be considered
 /// sufficient.
```

Comment:
Maybe let's add a clarification, now that we're using multiple different formulas, e.g.:

> This is the lowest weight so that any two sets of validators with that weight have at least one honest validator in common.

Reply:
Comment updated in 8a35e2cff0ceb9bdceea7e4e74f62e4a0eb0657b